tests/io/test_file.py

```diff
@@ -45,7 +45,7 @@ def test_open_file_read_lzma(self, tmpdir):
         with augur.io.file.open_file(path) as f_read:
             assert f_read.read() == 'foo\nbar\n'
 
-    def test_open_file_read_lzma(self, tmpdir):
+    def test_open_file_write_lzma(self, tmpdir):
```

Comment:
This meant the first function wasn't getting tested, yeah?

Reply:
Yup, I just noticed the number of tests that ran was one off when I was running the new zstd tests.